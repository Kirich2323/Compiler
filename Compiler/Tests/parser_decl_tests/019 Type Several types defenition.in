type
    myType = integer;
    myOtherType = integer;
    myAnotherType = myOtherType;
    myFourthType = float;
    myLastType = char;