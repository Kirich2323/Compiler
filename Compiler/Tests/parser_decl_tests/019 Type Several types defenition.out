main
----------
char                     type 
float                    type 
integer                  type 
myType             type alias         integer
myOtherType        type alias         integer
myAnotherType      type alias         integer
myFourthType       type alias           float
myLastType         type alias            char